skbio/stats/composition.py

```diff
@@ -806,6 +807,163 @@ def centralize(mat):
     cen = scipy.stats.gmean(mat, axis=0)
     return perturb_inv(mat, cen)
 
+@experimental(as_of="0.5.7")
+def vlr(x,y, ddof=1, robust=False):
+    r"""
+    Calculates variance log ratio
+    
+    Parameters
+    ----------
+    x : array_like, float
+       a 1-dimensional vector of proportions
+    y : array_like, float
+       a 1-dimensional vector of proportions
+        
+    ddof: int
+        degrees of freedom
+        
+    robust: bool
+        mask zeros at the cost of performance
+       
+    Returns
+    -------
+    float
+         variance log ratio value
+         
+    Examples
+    --------
+    # No zeros
+    >>> x = [1,2,3]
+    >>> y = [5,8,13]
+    >>> %timeit vlr(x,y)
+    # 21.7 µs ± 979 ns per loop (mean ± std. dev. of 7 runs, 10000 loops each)
```

Comment:
Let's remove the `timeit` statements from the docs -- the user doesn't need to know this. But we can keep this sort information under the thread in the pull request to keep track of the runtime of the functions.